package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "egghead-ui",
-  "version": "1.1.0",
+  "version": "1.4.0",
```

Comment:
I would recommend not versioning until after a PR is approved and merged into master; so a release is separate from the feature PR. Here is my recommended workflow - would be good to add to the project README if you agree:

### Make changes
1) checkout and pull latest from master
2) create feature branch and check it out
3) make changes
4) create PR
5) once PR is approved, merge into master

### Release new version
6) checkout `release` and pull latest from master
7) run `yarn release`
8) merge release into master

### Notify consumers
9) consumers can then run `yarn upgrade egghead-ui` in their projects

Although the checkout in step 6 and step 8 wouldn't be needed if you could release from master